packages/smooth_app/lib/pages/preferences/abstract_user_preferences.dart

```diff
@@ -68,12 +63,8 @@ abstract class AbstractUserPreferences {
   Widget getHeaderHelper(final bool? collapsed) => UserPreferencesListTile(
         title: getTitle(),
         subtitle: getSubtitle(),
-        isCompactTitle: isCompactTitle(),
-        icon: isCompactTitle()
-            ? SmoothAnimatedCollapseArrow(collapsed: collapsed!)
-            : collapsed != null
-                ? getForwardIcon()
-                : null,
+        isCompactTitle: false,
+        icon: collapsed != null ? getForwardIcon() : null,
```

Comment:
does this mean we loose the animation when opening and closing ?

Reply:
Correct. That's a personal choice, that I can rollback.

Reply:
I know you are not a fan of animations but I have to say I prefer them over plain changing which meight confuse the user